test-suite/misc/bug_14550.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+$coqc misc/bug_14550/bug_14550.v
+R=$?
+
+if [ $R == 0 ]; then
```

Comment:
```suggestion
if $coqc misc/bug_14550/bug_14550.v; then
```

Reply:
Note this is not consistent with the style used in the rest of `misc`.